Godeps/_workspace/src/github.com/emicklei/go-restful/swagger/model_builder.go

```diff
@@ -258,6 +271,7 @@ func (b modelBuilder) jsonSchemaType(modelName string) string {
 		"int":       "integer",
 		"int32":     "integer",
 		"int64":     "integer",
+                "uint64":    "integer",
```

Comment:
I don't know if this is github issue or not, but I think formatting is a bit off.